frontend/src/Interface/FarmDetailsInterface.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * @summary Interface for the main data file
+ * @description This interface defines how the main data file should be structured,
+ *              which values it takes and any calculation results. This can then
+ *              be exported as JSON.
+ * @author @GDamaso
+ */
```

Comment:
I personally like to line up all the information. But that's a me thing, not a mandatory thing 
```suggestion
 * @summary     Interface for the main data file
 * @description This interface defines how the main data file should be structured,
 *                        which values it takes and any calculation results. This can then
 *                        be exported as JSON.
 * @author         @GDamaso
 */
```